FluentTc/Locators/MoreOptionsHavingBuilder.cs

```diff
@@ -8,13 +8,15 @@ public interface IMoreOptionsHavingBuilder
         IMoreOptionsHavingBuilder AsPersonal();
         IMoreOptionsHavingBuilder WithComment(string comment);
         IMoreOptionsHavingBuilder OnBranch(string branchName);
+        IMoreOptionsHavingBuilder OnChange(long changeId);
```

Comment:
I would prefer to allow fluent syntax with the OnChange as well. Let me show what I mean:
```C#
connectedTc.RunBuildConfiguration(buildConfiguration => buildConfiguration.Name("FluentTc"), 
    p => p.Parameter(paramName, paramValue), 
    o => o.OnBranch(branchName)
        .OnChange(change => change.Id(changeId) )
);
```
Although I would accept the proposed version as well and we might add an overload later on. What do you think?